src/styles/antd-overrides/dropdown.scss

```diff
@@ -1,3 +1,6 @@
+.ant-dropdown {
+  top: 90px !important;
```

Comment:
nahhh haha. if we change the nav bar height, this will break right?

Think there'll be a different solution here.

Reply:
Oh yeah ofc, my bad. Fixed in a different spot `menu.scss:104`